Makefile

```diff
@@ -65,16 +65,16 @@ test:
 # This target is used to run Go tests that require docker runtime.
 # Some tests, like pipeline tests, need to have docker available to run.
 test-docker: build
-	KPT_E2E_BIN=$(GOBIN)/kpt go test -cover --tags=docker ./...
+	KPT_E2E_BIN=$(GOBIN)/kpt PATH=$(GOBIN):$(PATH) go test -cover --tags=docker ./...
 
 # KPT_E2E_UPDATE_EXPECTED=true (if expected output to be updated)
 # target to run e2e tests for "kpt fn render" command
 test-fn-render: build
-	KPT_E2E_BIN=$(GOBIN)/kpt go test -v --tags=docker --run=TestFnRender ./e2e/
+	KPT_E2E_BIN=$(GOBIN)/kpt PATH=$(GOBIN):$(PATH) go test -v --tags=docker --run=TestFnRender ./e2e/
 
 # target to run e2e tests for "kpt fn eval" command
 test-fn-eval: build
-	KPT_E2E_BIN=$(GOBIN)/kpt go test -v --tags=docker --run=TestFnEval ./e2e/
+	KPT_E2E_BIN=$(GOBIN)/kpt PATH=$(GOBIN):$(PATH) go test -v --tags=docker --run=TestFnEval ./e2e/
```

Comment:
Since `GOBIN` overrides the default path, you probably don't need to even specify `KPT_E2E_BIN` since default `kpt` should resolve to the one in GOBIN. Right ?

Reply:
One of the purposes for introducing `KPT_E2E_BIN` is giving us capability to specify kpt binary to use. However, sink the command in scripts will use kpt in `PATH` so I think we should keep consistent with it. I will remove `KPT_E2E_BIN`.